frontend/src/components/app/App.js

```diff
@@ -8,14 +8,19 @@ import {
   Routes,
   Route,
 } from "react-router-dom";
-
+import Navbar from '../navbar/NavBar';
 const App = () => {
     return (
+      <>
+        <h2>Farcebook</h2>
```

Comment:
Tom and I wondered if this should be structured differently, but it seems fine to use this like this right now 👍🏼